docs/api-reference/xviz-metadata-builder.md

```diff
@@ -213,8 +213,10 @@ special and cannot be overwritten.
 
 Special keys:
 
-- `start_time`
-- `end_time`
+- `start_time` (Number) - Start time for the log data
+- `end_time` (Number) - End time for the log data
+- `log_start_time` - (Number) - Start time of the log from which this was created
```

Comment:
Do we need both `start_time` and `log_start_time` still?  I think part of the reason we had these internally was because of the way the code was developed.  Where one was supposed to represent the full duration of the log itself, and the other window you requested (echoing back what you sent to start up the connection).

Reply:
Correct.  Do we no longer need this distinction?  It seems like it is still interesting though could use a better description of why we have both.

Our use case grew to include this because we started with a batch type workflow, but once we *could* stream, it became desirable to explore the adjacent data as well.

Having said that, we could remove it and leave it as a custom thing.  It is easy enough to spec that  you can ask for more time, but you just get an error if it is no available.

Reply:
Another angle is that these two names are just quite confusing.  It's not clear from reading them what the difference is.

Reply:
ok, I'll limit it to `start_time` and `end_time` as it's easier to add others later than remove them.